lib/srv/reexec.go

```diff
@@ -114,6 +120,19 @@ type UaccMetadata struct {
 	WtmpPath string `json:"wtmp_path,omitempty"`
 }
 
+func getPAMEnvironment(c *ExecCommand) *map[string]string {
```

Comment:
Move this logic to `lib/srv/ctx.go` in `getPAMConfig to consolidate all PAM logic in one place.

Reply:
Done